critiquebrainz/frontend/external/bookbrainz_db/redirects.py

```diff
@@ -0,0 +1,49 @@
+import uuid
+from brainzutils import cache
+import sqlalchemy
+import critiquebrainz.frontend.external.bookbrainz_db as db
+from critiquebrainz.frontend.external.bookbrainz_db import DEFAULT_CACHE_EXPIRATION
+
+
+def get_redirected_bbid(bbid: uuid.UUID) -> str:
```

Comment:
It would be good to have a test for this, both for a single level redirect, and for a recursive one

Reply:
I've added the tests